plugins/greeting/controller.js

```diff
@@ -12,15 +12,13 @@ function Greeting($scope, $http, $interval) {
 			} else if (hour >= 23 || hour < 4) {
 				greetingTime = "night";
 			}
-			var nextIndex = Math.floor(Math.random() * config.greeting[greetingTime].length);
```

Comment:
> /home/travis/build/evancohen/smart-mirror/plugins/greeting/controller.js
  17:5   error  'nextIndex' is not defined  no-undef
  18:50  error  'nextIndex' is not defined  no-undef